packages/web/src/components/nav/desktop/PlaylistLibrary/CollectionNavItem.tsx

```diff
@@ -182,28 +181,46 @@ export const CollectionNavItem = (props: CollectionNavItemProps) => {
           kind='library-playlist'
         >
           <LeftNavLink
+            asChild
             to={url}
             onClick={onClick}
             disabled={isDisabled}
             onDragEnter={handleDragEnter}
             onDragLeave={handleDragLeave}
             onMouseEnter={handleMouseEnter}
             onMouseLeave={handleMouseLeave}
-            className={styles.root}
           >
-            <span
-              className={cn(styles.content, {
-                [styles.level1]: level === 1
-              })}
+            <Flex
+              alignItems='center'
+              w='100%'
+              pl={level * spacing.l}
+              gap='xs'
+              css={{
+                position: 'relative'
+              }}
             >
-              {hasUpdate ? <PlaylistUpdateDot /> : null}
-              <span className={styles.collectionName}>{name}</span>
+              {hasUpdate ? (
+                <div css={{ position: 'absolute', left: '-12px' }}>
+                  <PlaylistUpdateDot />
+                </div>
+              ) : null}
+              <Text
+                size='s'
+                css={{
+                  '.droppableLinkActive &': {
```

Comment:
fun... i guess this is okay, though maybe we use avoid the var

Reply:
Yeah if/when we update droppable to use a render function and just pass the state to children, we can fix this.
I did update it to use `color.text.accent` though, as it's the equivalent semantic color